akka-actor-typed-tests/src/test/scala/akka/actor/typed/WatchSpec.scala

```diff
@@ -24,6 +25,12 @@ object WatchSpec {
       case (_, Stop) ⇒ Behaviors.stopped
     }
 
+  val mutableTerminatorBehavior = new MutableBehavior[Stop.type] {
+    override def onMessage(msg: Stop.type) = msg match {
+      case Stop ⇒ Behaviors.stopped
+    }
+  }
+
```

Comment:
this was a false lead, but I decided to keep the test anyway

Reply:
It doesn't seem to be used anywhere, though?

Reply:
removed